allennlp/models/archival.py

```diff
@@ -175,8 +175,10 @@ def load_archive(archive_file: str,
         logger.info(f"loading archive file {archive_file} from cache at {resolved_archive_file}")
 
     if os.path.isdir(resolved_archive_file):
+        loading_dir = True
```

Comment:
I would slightly prefer a more descriptive name like `loading_from_directory` or `archive_is_directory` or something like that. `loading_dir` sounds like it's the directory itself

Reply:
Done